docs/Swift.md

```diff
@@ -1,21 +1,19 @@
-Theos is able to compile [Swift](https://swift.org/) files, including using them alongside files in Objective-C and other languages. Currently, Theos supports Swift on macOS, iOS, and Linux. Instructions on installing the Swift toolchains on [iOS](https://github.com/theos/theos/wiki/Installation-iOS) and [Linux](https://github.com/theos/theos/wiki/Installation-Linux) can be found on their respective installation wiki pages.
+Currently, Theos is capable of compiling [Swift](https://swift.org/) files and supports using them in conjunction with files written in other languages, such as Objective-C. These capabilities are supported on macOS, iOS, and Linux. Instructions for installing the Swift toolchains on [iOS](Installation-iOS.md) and [Linux](Installation-Linux.md) can be found on their respective installation wiki pages.
```

Comment:
Same comment on oxford comma as above. Also, `Currently,` is probably not necessary.